prow/pj-on-kind.sh

```diff
@@ -25,29 +25,34 @@ function main() {
 
   # Generate PJ and Pod.
   mkpj "--config-path=${config}" "${job_config}" "--job=${job}" > "${PWD}/pj.yaml"
-  mkpod --build-id=snowflake "--prow-job=${PWD}/pj.yaml" --local "--out-dir=/output/${job}" > "${PWD}/pod.yaml"
+  mkpod --build-id=snowflake "--prow-job=${PWD}/pj.yaml" --local "--out-dir=${out_dir}" > "${PWD}/pod.yaml"
+
+  # Add any k8s resources that the pod depends on to the kind cluster here. (secrets, configmaps, etc.)
 
   # Deploy pod and watch.
   echo "Applying pod to the mkpod cluster. Configure kubectl for the mkpod cluster with:"
-  echo '>  export KUBECONFIG="$(kind get kubeconfig-path --name="mkpod")"'
+  echo '>  export KUBECONFIG="$(kind get kubeconfig-path --name=mkpod)"'
```

Comment:
slightly off topic but https://github.com/kubernetes-sigs/kind/issues/850 for future direction there